views/templates/admin/configuration.tpl

```diff
@@ -0,0 +1,194 @@
+						{l s='Up to %d free email per day' mod='expressmailing' sprintf=$broadcast_max_daily}
+						<b>&nbsp;-&nbsp;{l s='or' mod='expressmailing'}&nbsp;-&nbsp;</b>
+						<a id="em_bying_link_email2"{if $smarty_email_disabled} disabled="disabled"{/if}>{l s='Sign Up for a Premium Plan' mod='expressmailing' sprintf=$smarty_email_lowest_price}</a>
+						{if !empty($smarty_email_promotion)}
+							<span class="badge"><i class="icon-star"></i> <small>{l s='Discount' mod='expressmailing'}</small></span>
+						{/if}
+					</td>
+				</tr>
+				<tr>
+					<td class="colorcell"><div class="redcell"></div></td>
+					<td class="linkcell">{l s='Fax' mod='expressmailing'}</td>
+					<td class="commentcell">
+						{l s='Lowest price to Metropolitan France' mod='expressmailing'}
+						<b>&nbsp;-&nbsp;</b>
+						{capture assign="smarty_fax_lowest_price"}{$smarty_fax_lowest_price|escape:'htmlall':'UTF-8'|string_format:"%.3f"|replace:'0.':'0,'|rtrim:'0'}{/capture}
+						{capture assign="fprice"}{l s='From %s € per page' mod='expressmailing' sprintf=$smarty_fax_lowest_price}{/capture}
+						<a id="em_bying_link_fax2"{if $smarty_fax_disabled} disabled="disabled"{/if}>{$fprice|escape:'html':'UTF-8'}</a>
+						{if !empty($smarty_fax_promotion)}
+							<b>&nbsp;-&nbsp;</b>
+							<span class="badge"><i class="icon-star"></i> <small>{l s='Discount' mod='expressmailing'}</small></span>
+						{/if}
+					</td>
+				</tr>
+				<tr>
+					<td class="colorcell"><div class="bluecell"></div></td>
+					<td class="linkcell">{l s='Sms' mod='expressmailing'}</td>
+					<td class="commentcell">
+						{l s='Lowest price to Metropolitan France' mod='expressmailing'}
+						<b>&nbsp;-&nbsp;</b>
+						{capture assign="smarty_sms_lowest_price"}{$smarty_sms_lowest_price|escape:'htmlall':'UTF-8'|string_format:"%.3f"|replace:'0.':'0,'|rtrim:'0'}{/capture}
+						{capture assign="fprice"}{l s='From %s € per sms' mod='expressmailing' sprintf=$smarty_sms_lowest_price}{/capture}
+						<a id="em_bying_link_sms2"{if $smarty_sms_disabled} disabled="disabled"{/if}>{$fprice|escape:'html':'UTF-8'}</a>
+						{if !empty($smarty_sms_promotion)}
+							<b>&nbsp;-&nbsp;</b>
+							<span class="badge"><i class="icon-star"></i> <small>{l s='Discount' mod='expressmailing'}</small></span>
+						{/if}
+					</td>
+				</tr>
+			</table>
+		</div>
+	</div>
+</div>
+<div class="col-lg-4">
+	<div class="em-fixed-col-lg">
+		<div class="row row-margin-bottom intro-left-blocs">
+			<div class="col-lg-12 text-center">
+				<h2 class="text-uppercase" style="margin-top: 0px">{l s='Subscription' mod='expressmailing'}</h2>
+				<a href="index.php?controller=AdminMarketingInscription&token={Tools::getAdminTokenLite('AdminMarketingInscription')|escape:'html':'UTF-8'}" class="btn btn-primary btn-lg" >{l s='Open your account' mod='expressmailing'}</a>
+				<div style="padding: 1em">
+					<div style="font-size: 15px; margin-right: auto; margin-left: auto; display: inline; text-align: left">
```

Comment:
In all this file, you should really think to move the style definition in a CSS file.